nextcord/ui/modal.py

```diff
@@ -171,6 +171,11 @@ def add_item(self, item: Item) -> Modal:
         item: :class:`Item`
             The item to add to the modal.
 
+            .. note::
+
+                At the current time, Discord supports only Text Input items in Modals.
+                The current Discord Docs page for supported items is [here](https://discord.com/developers/docs/interactions/receiving-and-responding#interaction-response-object-modal).
```

Comment:
As mentioned [here](https://canary.discord.com/channels/881118111967883295/881124672056467456/1119316194730709092), I'm not sure why the nextcord documentation needs to be "dynamic" on what item types are supported. If an item type is not in nextcord, the user cannot make it until it is added - in which, the documentation should be updated to go along with it.